src/services/storageTypes.ts

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2018-2020 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+
+export enum StorageTypeTitle {
```

Comment:
Why 'StorageTypeTitle'?  I expected for 'StorageType'.

Reply:
The reason is not to be confused what storage type is. Whether it's a string retrieved from server ("async", "ephemeral" or "permanent"), or its representation in UD ("Asynchronous", "Ephemeral" or "Permanent"). Difference is subtle but it matters and causes bugs.

Reply:
ok
I explain my approach [here](https://github.com/che-incubator/che-dashboard-next/pull/83#discussion_r530206506).